Gemfile.lock

```diff
@@ -1,94 +1,94 @@
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       mail (>= 2.7.1)
-    actionmailer (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     aws-eventstream (1.1.0)
-    aws-partitions (1.353.0)
-    aws-sdk-core (3.104.3)
+    aws-partitions (1.399.0)
+    aws-sdk-core (3.109.3)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.36.0)
-      aws-sdk-core (~> 3, >= 3.99.0)
+    aws-sdk-kms (1.39.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.76.0)
-      aws-sdk-core (~> 3, >= 3.104.1)
+    aws-sdk-s3 (1.85.0)
+      aws-sdk-core (~> 3, >= 3.109.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.2.1)
+    aws-sigv4 (1.2.2)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.15)
-    better_errors (2.7.1)
+    bcrypt (3.1.16)
+    better_errors (2.9.1)
```

Comment:
### [better_errors](https://github.com/BetterErrors/better_errors)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.7.1 → 2.9.1

[_[change-log](https://github.com/BetterErrors/better_errors/releases), [source-code](https://github.com/BetterErrors/better_errors)_]





<details>
<summary>Commits</summary>

A change of **86** commits. See the full changes on [the compare page](https://github.com/BetterErrors/better_errors/compare/v2.7.1...v2.9.1).

These are the first 10 commits:
- (890c83f) [add hint for NoMethodError](https://github.com/BetterErrors/better_errors/commit/890c83f2a583d45513115d4f9a0c107fa68a6a10)
- (56340ab) [give hints for NameErrors](https://github.com/BetterErrors/better_errors/commit/56340abf89f7fda9c256fadb212f1b4231aa68f3)
- (081fa84) [more cautious checking for NoMethodError hint](https://github.com/BetterErrors/better_errors/commit/081fa8454913cc443d285b2bd247fbaf79f0e070)
- (5f2cdc6) [adds target=_blank to editor link, to work around CSP](https://github.com/BetterErrors/better_errors/commit/5f2cdc63c8feda6f24f845805165b468a5a8a5b2)
- (5960d9a) [Add a button the error page if the error is actionable.](https://github.com/BetterErrors/better_errors/commit/5960d9ae5fe2e11456e3ad8519bb3e84674b3ed3)
- (11427ba) [Add specs for actionable errors](https://github.com/BetterErrors/better_errors/commit/11427babd0d6891b996b496c528d15aa486f5334)
- (776a12e) [Improve layout of multiline exception messages](https://github.com/BetterErrors/better_errors/commit/776a12e5f8b29fe4059dd486da8f6c8aa47f29b3)
- (c65b243) [Improve backtrace when a BE template raises error](https://github.com/BetterErrors/better_errors/commit/c65b24335453ed1809b238bd33e9a8da5ad59964)
- (87cc62d) [Fix wording of context](https://github.com/BetterErrors/better_errors/commit/87cc62d7dfeef345a28cee3e68a867fffe41af1f)
- (448c9f2) [Merge pull request #465 from connorshea/actionable-errors](https://github.com/BetterErrors/better_errors/commit/448c9f2f109723d292723310ed950e7bafeaa47e)


</details>